client/my-sites/earn/ads/form-settings.jsx

```diff
@@ -413,7 +404,7 @@ class AdsFormSettings extends Component {
 							/>
 							<FormSettingExplanation>
 								{ translate(
-									'Adds a link to your privacy policy to the bottom of the CCPA notice popup (optional).'
+									'Adds a link to your privacy policy to the bottom of the sale opt-out notice popup (optional).'
```

Comment:
What's a "**sale** opt-out notice?" Did you mean "states?"

Reply:
I meant "sale". The word "sale" seems to be consistently used in the 3 documents provided by legal to be used across the settings page and the "do not sell" modal. 
I assumed it relates to the fact that personal information is considered being "sold" to advertisers, and the "do not sell" link copy says `Do Not Sell or Share My Personal Information`.